xml/windows-appendix.xml

```diff
@@ -48,7 +53,50 @@
 </screen>
  </sect1>
  <sect1 xml:id="windows-appendix-b">
-   <title>Appendix B: Upstream Projects</title>
+   <title>Appendix B: Troubleshooting Tips</title>
+   <para>
+     If you encounter installation or driver problems, the following tips may
+     be helpful:
+   </para>
+   <para>
+     To generate an installation log:
+   </para>
+<screen>
+msiexec /i C:\path\to\ses4win.msi /l*v log.txt
+</screen>
+   <para>
+     Driver loading issues my be identified through the Windows driver log:
+   </para>
+<screen>
+C:\Windows\inf\Setupapi.dev.log
+</screen>
+   <para>
+     Manual uninstallation is possible using:
+   </para>
+<screen>
+msiexec /x [C:\path\to\ses4win.msi|{GUID}]
+</screen>
+   <note>
+     <para>
+     The GUID can be found under <literal>HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall</literal>
```

Comment:
```suggestion
     The GUID can be found under <filename>HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall</filename>.

Reply:
This is a Windows registry path, so I'm not sure if literal or filename is more appropriate. I guess it just depends on how it is rendered. We can switch to filename and see if that works.